src/lib/axios.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { API_URL } from "../constants";
+
+export default axios.create({
```

Comment:
Repeated code. Can we merge this into one export?

Reply:
Initially I believed we needed two exports, one with request and response interceptors and a default one for routes that don't require authentication. I tried with only one export and everything still works fine so...this means the other one is redundant.